src/js/pie_exploded.js

```diff
@@ -0,0 +1,301 @@
+import * as d3 from 'd3';
+
+export class PieExploded {
+	constructor(container, data) {
+		let container_div;
+
+		if (typeof container === 'string') {
+			container_div = document.querySelector(container);
+		} else {
+			container_div = container;
+		}
+
+		container_div.innerHTML = '';
+
+		d3.select(container_div).attr('chart_type', 'PieExploded');
+		
+		container_div.classList.add('PieExploded');
+		container_div.classList.add('d3chart');
+		container_div.classList.add('chart_container');
+
+		this.container = d3.select(container_div);
+
+		// Declare the chart dimensions and margins.
+		const width = 700;
+		const height = 500;
+		const marginTop = 40;
+		const marginRight = 40;
+		const marginBottom = 110;
+		const marginLeft = 110;
+
+		// Create the svg container
+		const svg = this.container
+			.append('svg')
+			.attr('width', width)
+			.attr('height', height)
+			.attr('viewBox', [0, 0, width, height])
+			.attr('preserveAspectRatio', 'xMinYMin meet')
+			.attr('style', 'max-width: 100%; height: auto;');
+
+		// Plot parameters
+		let hole_percent = 0.4;
+		let radius =
+			Math.min(
+				width - marginLeft - marginRight,
+				height - marginTop - marginBottom
+			) / 2;
+
+		let pie_center = radius + marginLeft;
+		let exploded_offset = 80;
+		let text_right_offset = 50;
+		let text_left_offset = 100;
+		let minLabelSpacing = 16;
+		let comment_position = marginRight + 150;
+		let comment_height = height - marginBottom / 3;
+
+			
+		// Plot labels
+		let numbers_long = { M: ' million', G: ' billion', T: ' trillion' };
+		let unit = 'USD'; // TODO: make a parameter
+		let comment = 'Total: ';
```

Comment:
```suggestion
		let comment = 'Exposure of Assets to PACTA Sectors: ';
```
I would rather not merge the "Total" caption in, since we know that is confusing. 
Let's use this for now as it is at least accurate, and then Nayra can validate all these things after a more in-depth review. 

Relates to #51

Reply:
I expect that the plot will have a descriptive title in which case I am not sure if 'Total' is not enough. Also, in case of emissions I don't think we can call it 'exposure'?